source/shared/cpp/ObjectModel/HostConfig.cpp

```diff
@@ -75,6 +84,24 @@ HostConfig HostConfig::Deserialize(const Json::Value& json)
     return result;
 }
 
+FontSizesConfig::FontSizesConfig()
+{
+    smallFontSize = UINT_MAX;
+    defaultFontSize = UINT_MAX;
+    mediumFontSize = UINT_MAX;
+    largeFontSize = UINT_MAX;
+    extraLargeFontSize = UINT_MAX;
+}
+
+FontSizesConfig::FontSizesConfig(unsigned int s, unsigned int d, unsigned int m, unsigned int l, unsigned int xl)
+{
+    smallFontSize = s;
+    defaultFontSize = d;
+    mediumFontSize = m;
+    largeFontSize = l;
+    extraLargeFontSize = xl;
+}
```

Comment:
members should be initialized using a member initializer list: [https://en.cppreference.com/w/cpp/language/initializer_list](https://en.cppreference.com/w/cpp/language/initializer_list)

Reply:
(same throughout)

---
In reply to: [227140870](https://github.com/Microsoft/AdaptiveCards/pull/2101#discussion_r227140870) [](ancestors = 227140870)